chart/kubeapps/values.yaml

```diff
@@ -23,6 +23,8 @@ apprepository:
     url: https://kubernetes-charts-incubator.storage.googleapis.com
   - name: svc-cat
     url: https://svc-catalog-charts.storage.googleapis.com
+  - name: bitnami
```

Comment:
Can we add this in another PR? It does not seem to be related to the rest of the PR.

Reply:
Thank you!